tests/bundle/Integration/BaseTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\IbexaScheduledVisibility\Tests\Integration;
+
+use DateTime;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType;
+use Ibexa\Tests\Integration\Core\Repository\BaseTest as APIBaseTest;
+use Netgen\Bundle\IbexaScheduledVisibilityBundle\Service\ScheduledVisibilityService;
+
+abstract class BaseTest extends APIBaseTest
```

Comment:
You can try to configure GitHub actions to run the tests automatically, for example see https://github.com/netgen/ibexa-site-api/blob/master/.github/workflows/tests.yml

Reply:
added here fe70e4ecc62224d012cd3a8455e6e024c82f981b. This has been taken from tags bundle as it seemed to me to be compatible.